feature/velero/install.sh

```diff
@@ -3,10 +3,10 @@
 TEMPDIR=$(mktemp -d)
 cd "${TEMPDIR}" || exit
 
-execHandle "Downloading velero" curl -f -s -L "https://github.com/vmware-tanzu/velero/releases/download/${VELERO_VERSION}/velero-${VELERO_VERSION}-linux-amd64.tar.gz" --output velero.tar.gz
+execHandle "Downloading velero" curl -f -s -L "https://github.com/vmware-tanzu/velero/releases/download/v${VELERO_VERSION}/velero-v${VELERO_VERSION}-linux-$(getPlatform).tar.gz" --output velero.tar.gz
```

Comment:
This would be a BREAKING_CHANGE because it require a change in configuration for users

Reply:
IMHO I would remove this, because while I can understand the idea here, it introduces a difference to the referenced projects versioning scheme. If a project chooses to use the "v", it's the official release name on e.g. Github and should be referenced for CloudControl.

Reply:
And I just saw that other features apparently already do this. :/

So yes, we should decide to either go the one or the other way, but this will result in a BREAKING CHANGE.

Reply:
yes i know, but i find it VERY confusing using "v1.2.3" on one feature and "1.2.3" on the other feature. I would rather keep having all with just the version number to have it equal and let the feature install script take care of the details.

If that results in a breaking change, well then it does 🤷‍♂️